ui_components/build/travel/Travel.js

```diff
@@ -544,9 +544,7 @@ function TravelActions({
     onPointerUp: handlePointerUp,
     onPointerLeave: handlePointerUp,
     onContextMenu: e => {
-      if (e.nativeEvent.pointerType === "touch") {
-        e.preventDefault();
-      }
+      e.preventDefault();
```

Comment:
Ah, why was this needed?

Reply:
People were reporting that when holding down movement keys on mobile the context menu is popping up after a few seconds. For some reason the pointerType === "touch" check isn't inclusive enough so I'm trying the nuclear option.